src/style.css

```
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Bold;
```

```suggestion
@font-face {
  font-family: Avenir;
  src: url(fonts/Avenir-Book.ttf);
}

@font-face {
  font-family: Avenir;
```
You need to use only one font-family but with different font-weights